addons/website_sale_product_configurator/static/src/js/website_sale_options.js

```diff
@@ -98,7 +98,7 @@ publicWidget.registry.WebsiteSale.include({
                 'item_name': el.getElementsByClassName('product_display_name')[0].textContent,
                 'quantity': parseFloat(el.getElementsByClassName('js_quantity')[0].value),
                 'currency': currency,
-                'price': parseFloat(el.getElementsByClassName('oe_price')[0].getElementsByClassName('oe_currency_value')[0].textContent),
+                'price': parseFloat(el.getElementsByClassName('js_raw_price')[0].value),
```

Comment:
The element in not an input so it doesn't have a `value`. Could you use `textContent` instead?
```suggestion
                'price': parseFloat(el.getElementsByClassName('js_raw_price')[0].textContent),
```